network-config-analyzer/K8sPolicyYamlParser.py

```diff
@@ -422,14 +422,25 @@ def parse_policy(self):
         res_policy.selected_peers &= self.peer_container.get_namespace_pods(self.namespace)
 
         ingress_rules = policy_spec.get('ingress', [])
+        exception = ''
         if ingress_rules:
             for ingress_rule in ingress_rules:
-                res_policy.add_ingress_rule(self.parse_ingress_rule(ingress_rule, res_policy.selected_peers))
+                try:
+                    res_policy.add_ingress_rule(self.parse_ingress_rule(ingress_rule, res_policy.selected_peers))
+                except ValueError as e:
+                    exception = e.args
+                if exception:
+                    self.syntax_error(exception)
```

Comment:
```suggestion
                except ValueError as e:
                    self.syntax_error(e.args)
```

Reply:
Initially I implemented it like you suggested, and then I saw in the output two exceptions, and the message that exception was thrown while handling another exception. I found the solution for this - added 'from None' when raising syntax_error exception.